### DISCUSSION TITLE
Exact meaning of LEDs

### DISCUSSION THREAD

**Discussion by camueller** [meatpiHQ/wican-fw#102] [General]
I was unable to find documentation about the exact meaning of the LEDs.

For instance, while driving, the blue power LED is sometimes blinking (like going on every 5s for about half a second) and sometimes it stays on all the time.

The Wifi LEDs seems to be indicating just the connection state.

The third LED with the letter sign I have no clue about its meaning. I have seen it being on just 2 or 3 times but don't know the reason.

**Comment by camueller** [comment]
Just returned from another drive: Blue LED was blinking and remained that way when I reached my home and turned off the car. Wifi LED did not turn on and hence the MQTT server did not receive the status message.

This should not happen, right?

**Reply by meatpiHQ** [reply]
@camueller do you have sleep mode enabled?

When the Blue LED is blinking this means it's in sleep mode.

Blue LED: Power LED
Green LED: connection LED, MQTT, TCP or Bluetooth 
Orange LED: CAN RX LED.

**Reply by camueller** [reply]
Yes, I have sleep mode enabled. But sleep mode should not be entered while I'm driving with the car! According to the [documentation](https://github.com/meatpiHQ/wican-fw?tab=readme-ov-file#1-sleep-mode) this should not happen.

**Reply by meatpiHQ** [reply]
What is the sleep voltage threshold set in your configuration?

**Reply by camueller** [reply]
I did not change the value - it's the default value of 13.1V.  If I turn off the car the voltage drops from 14.4V to 12.0V (according to WiCAN status). Is it possible that the voltage sometimes is not read correctly?

BTW: The 12V battery in the cat is less than a year old.

**Reply by meatpiHQ** [reply]
There's a possible bug in voltage reading that I'm already looking into. I'll send you a new firmware when it's ready. Until then please try to set a higher threshold.

**Reply by camueller** [reply]
Thank you for working on this issue!

> Until then please try to set a higher threshold.

Shouldn't I set it to a *lower* threshold in order to avoid going into sleep mode if the car is driving?

**Reply by camueller** [reply]
Since I set the sleep voltage threshold to 12.8 V sleep mode has never been entered while driving. As a result WiFi detection and MQTT work perfectly.

**Reply by meatpiHQ** [reply]
Good to hear. But I think there is still an issue with voltage reading.
